jax/experimental/jax2tf/examples/tflite/mnist/README.md

```diff
@@ -0,0 +1,92 @@
+# MNIST classification
+
+This directory contains the code necessary to train a simple convolution network on the MNIST dataset, and subsequently convert it using jax2tf into a model that can be converted to tflite format.
```

Comment:
Suggestion (just making the complex sentence more accessible for readability):

```suggestion
This directory contains the code that demonstrates how to:

1. Train a simple convolutional neural network on the MNIST dataset; and
2. Use jax2tf — that helps stage JAX programs out as TensorFlow graphs — into a TensorFlow [SavedModel](https://www.tensorflow.org/guide/saved_model) that can be used again for conversion into a [TensorFlow Lite (TF Lite)](https://www.tensorflow.org/lite/) format for on-device machine learning inference.
```

@qiuminxu Let me know if I got this gist of this right 😃 (especially the SavedModel part)